### ISSUE TITLE
🖐️ dagger install fails

### ISSUE THREAD

**Issue by wvell** [dagger/dagger#8291]
### What happened? What did you expect to happen?

I'm probably doing something wrong but i cant seem to figure out what is going wrong.
When i try to install the example from dagger install --help i get the following output:

    dagger install github.com/shykes/hello@v0.1.0

```
Output:
✘ moduleSource(refString: "github.com/shykes/hello@v0.1.0"): ModuleSource! 0.0s
! failed to resolve git tags: select: git command failed: exit status 128
Error: failed to get module ref kind: input: moduleSource resolve: failed to resolve git tags: select: git command failed: exit status 128
stdout:
stderr: fatal: 'github.com/shykes/hello' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

```

```
Version git:
git version 2.43.0

Version dagger:
dagger v0.12.6 (registry.dagger.io/engine:v0.12.6) linux/amd64
```

**Comment by jochumdev** [comment]
for me dagger install works when i prepend the url with the https scheme:

```

**Comment by wvell** [comment]
Thanks a lot. That fixes it for me :)

**Comment by jedevc** [comment]
Hm, this definitely used to work. This is likely a regression introduced by #7708 - cc @grouville.

Leaving off the version tag also seems to cause this to work, but we should fix this asap.

**Comment by jedevc** [comment]
Prepping a patch release today for this: https://github.com/dagger/dagger/pull/8299 - sorry for the inconvenience y'all :heart: 

As a workaround, yup, you can manually use `https://`.

**Comment by jedevc** [comment]
This is now fixed in v0.12.7. However, note, the command `dagger install github.com/shykes/hello@v0.1.0` will not work.

This module version number is out of date (and the docs are as well for including it). As an example hello world module, I recommend using `dagger install github.com/shykes/daggerverse/hello@v0.3.0` (and I've also updated the docs to use this as well) :tada:

**Comment by pjmagee** [comment]
`dagger install github.com/shkyes/daggerverse/hello@v0.3.0` doesn't work for me. :( 

```
mageep@MacBook-Air dagger-private-hello-world % dagger install github.com/shkyes/daggerverse/hello@v0.3.0
Setup tracing at https://dagger.cloud/traces/setup. To hide: export STOPIT=1

✔ connect 0.7s
✔ cache request: mkfile /schema.json 0.0s
✔ load cache: mkfile /schema.json 0.0s
✔ cache request: blob://sha256:1d4c085c7e5b11f0444448c9bbced6d47f0f4c1a0a7ee6d0ca3d61e58bb8400c 0.0s
✔ load cache: blob://sha256:1d4c085c7e5b11f0444448c9bbced6d47f0f4c1a0a7ee6d0ca3d61e58bb8400c 0.0s
✔ moduleSource(refString: "."): ModuleSource! 0.0s
✔ ModuleSource.kind: ModuleSourceKind! 0.0s
✔ ModuleSource.resolveContextPathFromCaller: String! 0.0s
✘ moduleSource(refString: "github.com/shkyes/daggerverse/hello@v0.3.0"): ModuleSource! 0.1s
! failed to resolve git tags: select: git command failed: exit status 128
Error: failed to get module ref kind: input: moduleSource resolve: failed to resolve git tags: select: git command failed: exit status 128
stdout: 
stderr: fatal: could not read Username for 'https://github.com': No such device or address
```

```
mageep@MacBook-Air dagger-private-hello-world % dagger version
dagger v0.12.7 (registry.dagger.io/engine:v0.12.7) darwin/arm64
```

**Comment by pjmagee** [comment]
> This is now fixed in v0.12.7. However, note, the command `dagger install github.com/shykes/hello@v0.1.0` will not work.
> 
> This module version number is out of date (and the docs are as well for including it). As an example hello world module, I recommend using `dagger install github.com/shykes/daggerverse/hello@v0.3.0` (and I've also updated the docs to use this as well) 🎉

this works for me, thank you
